Library/Homebrew/cleanup.rb

```diff
@@ -31,7 +31,6 @@ def self.cleanup_formula(formula)
     end
 
     def self.cleanup_logs
-      return unless HOMEBREW_LOGS.directory?
```

Comment:
Is this indeed no longer needed? This looks like a leftover from the original implementation of this PR.